tensorflow/workspace.bzl

```diff
@@ -35,6 +36,7 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   nccl_configure(name="local_config_nccl")
   git_configure(name="local_config_git")
   sycl_configure(name="local_config_sycl")
+  rocm_configure(name="local_config_rocm")
```

Comment:
You probably need to add an exclude to build_pip_package.sh for this too.
here: https://github.com/tensorflow/tensorflow/blob/622a111e88dfbac430556ec8213230345189f466/tensorflow/tools/pip_package/build_pip_package.sh#L30